starkinfra/issuingtokenactivation/__issuingtokenactivation.py

```diff
@@ -0,0 +1,44 @@
+from ..utils.parse import parse_and_verify
+from starkcore.utils.subresource import SubResource
+
+
+class IssuingTokenActivation(SubResource):
+    """# IssuingTokenActivation object
+    The IssuingTokenActivation object displays the necessary information to proceed with the card tokenization.
```

Comment:
You will receive this object at your registered URL to notify you which method your user want to receive the activation code.
The POST request must be answered with no body, within 2 seconds, and with an HTTP status code 200.
After that, you may generate the activation code and send it to the cardholder.